routes/root.js

```diff
@@ -0,0 +1,26 @@
+const router = require("express").Router();
+const path = require("path");
+const {
+  getVacations,
+  employeeInfo,
+  homeAddress,
+  askVacation,
+} = require("./../controllers/employeeDashboard.controller");
+
+router.get("^/$|index(.html)?", (req, res) => {
+  res.sendFile(path.join(__dirname, "..", "client", "index.html"));
+});
+
+router.get("/api/employeeInfo", (req, res) => {
+  employeeInfo(req, res);
+});
+
+router.put("/api/home_address", (req, res) => {
+  homeAddress(req, res);
+});
+
+router.post("/api/vacation_request", (req, res) => {
+  askVacation(req, res);
+});
+
+module.exports = router;

```

Comment:
your routes dont come here. open a new file with the name user.routes, and move everything you have added to this document there. Your routes should be called like so:
app.get /api/user 
app.put /api/user:address

vacation request is not your rout, so delete it.

after you do that, add the routes to the index.js and check everything with post man